charts/lnd/templates/_helpers.tpl

```diff
@@ -60,3 +60,19 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{- define "walletPassword" -}}
+
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace ( printf "%s-pass" (include "lnd.fullname" .))) -}}
+{{- if $secret -}}
+{{/*
+   Reusing current password since secret exists
+*/}}
+{{-  $secret.data.password -}}
```

Comment:
this is the syntax to return a value?

Reply:
It's the syntax to define a value, yes. `$secret` just refers to the result from the lookup.